dask/dataframe/io/tests/test_sql.py::test_empty failing with sqlalchemy.exc.RemovedIn20Warning

sqlalchmey 1.4.46 released Jan 3rd throws RemovedIn20Warning in dask/dataframe/io/tests/test_sql.py::test_empty  https://docs.sqlalchemy.org/en/20/changelog/changelog_14.html#change-e67bfa1efbe52ae40aa842124bc40c51

```
db = 'sqlite:////tmp/tmp1ems8js6'

    def test_empty(db):
        from sqlalchemy import Column, Integer, MetaData, Table, create_engine
    
        with tmpfile() as f:
            uri = "sqlite:///%s" % f
            metadata = MetaData()
            engine = create_engine(uri)
            table = Table(
                "empty_table",
                metadata,
                Column("id", Integer, primary_key=True),
                Column("col2", Integer),
            )
            metadata.create_all(engine)
    
>           dask_df = read_sql_table(table.name, uri, index_col="id", npartitions=1)

dask/dataframe/io/tests/test_sql.py:59: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
dask/dataframe/io/sql.py:301: in read_sql_table
    table_name = sa.Table(
<string>:2: in __new__
    ???
../../anaconda3/envs/dask-311/lib/python3.11/site-packages/sqlalchemy/util/deprecations.py:369: in warned
    _warn_with_version(
../../anaconda3/envs/dask-311/lib/python3.11/site-packages/sqlalchemy/util/deprecations.py:47: in _warn_with_version
    _emit_uber_warning(type_, stacklevel)
../../anaconda3/envs/dask-311/lib/python3.11/site-packages/sqlalchemy/util/deprecations.py:105: in _emit_uber_warning
    _warnings_warn(warn, stacklevel=stacklevel + 1)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

message = RemovedIn20Warning('Deprecated API features detected! These feature(s) are not compatible with SQLAlchemy 2.0. To prev...o show all deprecation warnings.  Set environment variable SQLALCHEMY_SILENCE_UBER_WARNING=1 to silence this message.'), category = None
stacklevel = 5

    def _warnings_warn(message, category=None, stacklevel=2):
    
        # adjust the given stacklevel to be outside of SQLAlchemy
        try:
            frame = sys._getframe(stacklevel)
        except ValueError:
            # being called from less than 3 (or given) stacklevels, weird,
            # but don't crash
            stacklevel = 0
        except:
            # _getframe() doesn't work, weird interpreter issue, weird,
            # ok, but don't crash
            stacklevel = 0
        else:
            # using __name__ here requires that we have __name__ in the
            # __globals__ of the decorated string functions we make also.
            # we generate this using {"__name__": fn.__module__}
            while frame is not None and re.match(
                r"^(?:sqlalchemy\.|alembic\.)", frame.f_globals.get("__name__", "")
            ):
                frame = frame.f_back
                stacklevel += 1
    
        if category is not None:
            warnings.warn(message, category, stacklevel=stacklevel + 1)
        else:
>           warnings.warn(message, stacklevel=stacklevel + 1)
E           sqlalchemy.exc.RemovedIn20Warning: Deprecated API features detected! These feature(s) are not compatible with SQLAlchemy 2.0. To prevent incompatible upgrades prior to updating applications, ensure requirements files are pinned to "sqlalchemy<2.0". Set environment variable SQLALCHEMY_WARN_20=1 to show all deprecation warnings.  Set environment variable SQLALCHEMY_SILENCE_UBER_WARNING=1 to silence this message. (Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9)

../../anaconda3/envs/dask-311/lib/python3.11/site-packages/sqlalchemy/util/langhelpers.py:1679: RemovedIn20Warning
```

--- a/setup.cfg
+++ b/setup.cfg
@@ -74,6 +74,7 @@ filterwarnings =
     # Ignore FutureWarning for change in group_keys behavior: no changes required on the dask side.
     # https://pandas.pydata.org/docs/dev/whatsnew/v1.5.0.html#using-group-keys-with-transformers-in-groupby-apply
     ignore:Not prepending group keys:FutureWarning
+    ignore:.*:sqlalchemy.exc.RemovedIn20Warning
 xfail_strict=true
 
 [metadata]

